lib/api.ts

```diff
@@ -53,132 +22,159 @@ export interface MemberPublicEditing extends MemberPublic {
   editing?: { field: string; changeTo: string | string[] }[];
 }
 
-export async function getMembers(): Promise<MemberPublic[]> {
-  return Promise.all([
-    getBase({ name: "Members", view: "Approved" }),
-    getBase({ name: "Focuses", view: "Approved" }),
-    getBase({ name: "Industries", view: "Approved" }),
-    getBase({ name: "Regions" }),
-  ]).then((values) => {
-    const [members, focuses, industries, regions] = values;
-    return members
-      .map((member) => {
-        const regionLookup =
-          regions.find((region) => {
-            const memberRegion = member.fields["Region"];
-            if (memberRegion && Array.isArray(memberRegion)) {
-              return region.id === member.fields["Region"][0];
-            }
-          })?.fields["Name"] || null;
+export interface DocumentData {
+  id: string;
+  fields: any;
```

Comment:
Might be a hot take, but it's probably easier to maintain strongly typed objects and use a data converter https://firebase.google.com/docs/reference/js/firestore_.firestoredataconverter rather than littering typechecks throughout the codebase.

Reply:
ohhh I never know about this, no yea I don't even see that as a hot take, just makes more sense

yea I think this should at least be implemented in the next PR if not this one